src/components/login_form.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component, PropTypes } from 'react';
+import { reduxForm } from 'redux-form';
+import Link from 'react-router';
+
+import { createAuth, loginAuth } from '../actions/index';
+
+class LoginForm extends Component {
+  componentWillMount() {
+    //  create auth and put in props
+    this.props.createAuth();
+  }
+
+  static contextTypes = {
+    router: PropTypes.object
+  };
+
+  onSubmit(input) {
+    const { auth } = this.props;
+
+    this.props.loginAuth(input, auth);
+  }
+
+  redirect() {
+    const { auth, login } = this.props;
+
+    // save login in localStorage before redirection
+    localStorage.setItem('login', JSON.stringify(login));
+    // sets a cookie on redirect
+    auth.session.setCookieAndRedirect(login.res.sessionToken, auth.options.redirectUri + 'home'); 
```

Comment:
Let's talk a little more about the flow we want to show - I don't think we should actually need to do a redirect in the SPA examples - only need the redirect if we care if Okta has a session (and at that point, we're usually redirecting to the interstitial page at the time we're logging into a downstream app).